src/server/on-off-farm/controller.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * The first question to start the journey.
+ * @satisfies {Partial<ServerRoute>}
+ */
+export const onOffFarmGetController = {
+  handler(req, h) {
+    req.yar.set('animal', 'cow')
+    const onOffFarm = req.yar.get('onOffFarm')
```

Comment:
Is this not asynchronous? If we're doing a round-trip to redis, then I'd want to be awaiting here.

Reply:
i thought the same - but apparently not looking at the source
I suspect what _actually_ happens is that it stores it in memory, then flushes to REDIS as a persistence type thing
so REDIS is infact a backup almost